templates/parse_file.html

```diff
@@ -54,8 +54,8 @@ <h1 class="title">CAMELOT Parse file {{filename}}</h1>
                 {% for column_name,best_column_name in zip(table.colnames, best_column_names) %}
                 <tr>
                     <div class="form-group">
-                    <td><p>{{ column_name }}</p></td>
-                    <td>
+                    <td style="vertical-align:middle"><p>{{ column_name }}</p></td>
+                    <td style="vertical-align:middle">
```

Comment:
Instead of hard-coding the "vertical-align: middle", make this a class:
`<td class="middle">`
then put in the css:
`.middle {vertical-align: middle;}`
or
`td.middle {vertical-align: middle;}`